### PR TITLE
New repo: cloud-provider-nutanix

### PR DESCRIPTION
**Pull request by Fedosin**
This PR sets up CI for a new forked repo: https://github.com/openshift/cloud-provider-nutanix

### CODE REVIEW

File: ci-operator/config/openshift/cloud-provider-nutanix/OWNERS [hunk 1]
```diff
@@ -0,0 +1,8 @@
+approvers:
+- Fedosin
+- JoelSpeed
+- elmiko
+- lobziik
+
+component: "Cloud Compute"
+subcomponent: "Cloud Controller Manager"
```
Comment (elmiko) [inline ci-operator/config/openshift/cloud-provider-nutanix/OWNERS:8]: i know this is a CCM, but we have a Nutanix subcomponent now, i wonder if we should use that instead?

**Review by elmiko — COMMENTED** [review]
looks mostly good, but i wonder if we should use the Nutanix subcomponent?

**Comment by lobziik** [comment]
Have the same question as Mike.

Aside this, lgtm

File: ci-operator/config/openshift/cloud-provider-nutanix/openshift-cloud-provider-nutanix-main.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      namespace: ocp
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: e2e-aws
```
Comment (lobziik) [inline ci-operator/config/openshift/cloud-provider-nutanix/openshift-cloud-provider-nutanix-main.yaml:46]: Maybe we could run linter and unittests instead?

Don't think that it worth to run aws e2e there now...

Comment (JoelSpeed) [reply ci-operator/config/openshift/cloud-provider-nutanix/openshift-cloud-provider-nutanix-main.yaml:46]: You need to have some sort of E2E before adding this to payload, is there a nutanix E2E?

File: core-services/prow/02_config/openshift/cloud-provider-nutanix/_pluginconfig.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+approve:
+- commandHelpLink: ""
+  repos:
+  - openshift/cloud-provider-nutanix
+  require_self_approval: false
```
Comment (JoelSpeed) [inline core-services/prow/02_config/openshift/cloud-provider-nutanix/_pluginconfig.yaml:5]: We normally have this true

File: core-services/prow/02_config/openshift/cloud-provider-nutanix/_pluginconfig.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+lgtm:
+- repos:
+  - openshift/cloud-provider-nutanix
+  review_acts_as_lgtm: true
```
Comment (JoelSpeed) [inline core-services/prow/02_config/openshift/cloud-provider-nutanix/_pluginconfig.yaml:9]: We normally have this false

File: core-services/prow/02_config/openshift/cloud-provider-nutanix/_pluginconfig.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+approve:
```
Comment (JoelSpeed) [inline core-services/prow/02_config/openshift/cloud-provider-nutanix/_pluginconfig.yaml:1]: You need to set up the backport risk assessed and cherry pick approval labels here

**Review by JoelSpeed — COMMENTED** [review]
/lgtm
/approve

**Comment by JoelSpeed** [comment]
Do we want to add any additional tests to this? Eg basic testing and linting?

**Comment by bparees** [comment]
image naming matches conventions

/approve

**Pull request merged**